_includes/icon-slack.html

```diff
@@ -0,0 +1 @@
+<a href="https://join.slack.com/t/catholicdevs/shared_invite/zt-dz50teya-019q4Zake3JdmsWiTxRBBg"><i class="fab fa-slack"></i> <span class="username">catholicdevs</span></a>
```

Comment:
@mkasberg @JohnRDOrazio just make sure this is invitation link is not going to expire.

I faced some issues with this when the slack was starting. I think when @JohnRDOrazio joined, he asked for a new link. Not sure, though.

Can you guys please check? 🤠

Reply:
I don't think I had any trouble with links expiring, I don't remember this?

However when I click on "invite friends -> copy invitation link" inside of the Slack interface, it says "link will expire in more than 30 days". So a link can expire after a while. In fact there is an "invitation settings" option next to "copy invitation link" which allows you to set the expiration to 1 day, 7 days or 30 days. So yes, links can expire.

We would probably need a way for people to send a request to join, rather than join using a link. Wherever the requests wind up, whoever sees and receives the request would then be able to send an invitation to the person requesting using the email address that was provided by the person requesting. Perhaps if we have an email account that can deal with this? Either that or it's writing some code to create a request process using the slack APIs (see near the bottom of [this page](https://slack.com/intl/en-va/help/articles/115004854783-Manage-workspace-invitation-permissions-and-requests)), but that starts to become a bit more work, not sure if it's worth it at this point in time.

Reply:
Oh, I didn't realize that link expires after 30 days.

I'll figure out some kind of workflow we can use here. I think there are some automated options available. I'll look into it and get back.

Reply:
I came up with a few options. We can discuss more here or on Slack if you want.

I think the easiest option is to try this, which looks like a free service that does just what we want. https://communityinviter.com/

An alternative option (harder, but probably not too bad) is I could implement an API in Heroku or AWS Lambda to generate invitations using the Slack API. Something like https://github.com/awskrug/slack-invite or https://github.com/outsideris/slack-invite-automation.

@JohnRDOrazio Let me know your thoughts!